### DISCUSSION TITLE
主题：Blue Topaz，蓝色托帕石

### DISCUSSION THREAD

**Discussion by whyt-byte** [obsidianzh/forum#22] [General综合]
## 目前长这样 （版本v1.7，编号20201209）：
![](https://github.com/whyt-byte/Blue-Topaz_Obsidian-css/blob/master/preview_Blue%20Topaz.png?raw=true)

介绍来说的话，一个蓝色主题

### 主要特点
1. 多彩标签
2. 多彩笔记框（使用方法见下方Github地址或css文件开头TIPS）
3. 记忆笔记框，可点击显示隐藏文字
4. 很棒的列表格式（改自Gabroel在Obsidian discord group上发出的代码）
5. 常用插件修改以适配主题配色

### 如何获取
- 社区主题内搜索：Blue Topaz
- Github地址在这：https://github.com/whyt-byte/Blue-Topaz_Obsidian-css

### 欢迎反馈各种bug

**Comment by makerjackie** [comment]
赞！

**Reply by makerjackie** [reply]
@whyt-byte 有bug
![image](https://user-images.githubusercontent.com/18050469/101774820-9128cc00-3b29-11eb-97e2-35c297d99022.png)

**Reply by makerjackie** [reply]
另外：
![image](https://user-images.githubusercontent.com/18050469/101775089-efee4580-3b29-11eb-8006-7b92a06eb967.png)

**Reply by whyt-byte** [reply]
嗯，是目录分栏。

bug之前发现过：在某些宽度时会出现抖动或匹配不当。

介于基本上没人拉太宽，分栏作用不大，最近在考虑删除分栏。

**Reply by makerjackie** [reply]
挺棒的颜色主题，如果修复这个bug我就使用啦！感谢！

**Reply by whyt-byte** [reply]
修了，社区主题更新就行
